node/sc/bridge_accounts.go

```diff
@@ -37,6 +37,11 @@ import (
 
 const (
 	DefaultBridgeTxGasLimit = 5000000
+
+	ParentOperatorStr              = "parentOperator"
+	ChildOperatorStr               = "childOperator"
+	DefaultParentBridgeAccountName = "parent_bridge_account"
```

Comment:
Why do we need 'Default' prefix here?

Reply:
This is because hardcoded meaning that the file name is always immutable.

Reply:
So, can we change this to another name? Because it is default, we can use non-default one if we want?

Reply:
To do that, we need more work such as taking the directory name the user wants and then do change with that name. Also, the constant keyword should be removed. I think it's better for this work to move to another PR.

Reply:
@hyunsooda I don't get the point. Are those file names? You said that it's immutable, then I think we can use `ParentBridgeAccountName` instead of `DefaultParentBridgeAccountName`.

Reply:
Now I see. I overly thought for futures to be considered. Removed default prefix.